packages/circuit-ui/components/Anchor/Anchor.tsx

```diff
@@ -40,8 +46,8 @@ export interface BaseProps extends BodyProps {
    */
   ref?: Ref<any>;
 }
-type LinkElProps = Omit<HTMLProps<HTMLAnchorElement>, 'size' | 'onClick'>;
-type ButtonElProps = Omit<HTMLProps<HTMLButtonElement>, 'size' | 'onClick'>;
+type LinkElProps = Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'onClick'>;
+type ButtonElProps = Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onClick'>;
```

Comment:
What motivated this change? Is it something like https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36505#issuecomment-549394273?

Reply:
Yes, something like this! TS was unhappy with a few things after the migration around HTML element props, so I dug into the React types.

- `HTMLProps` actually include a lot more than just the valid element attributes, that's why we often had to omit the `size` prop for example, although it's not a valid HTML attribute (on most elements)
- `*HTMLAttributes` extends `HTMLAttributes` if there are any extra attributes available for the `*` element. So for example there will be a `ButtonHTMLAttributes` type, but a `div` will simply use `HTMLAttributes`

If you know/find more info on this I'm happy to revisit it, but as far as I could tell this is the most accurate solution :slightly_smiling_face: